src/breadcrumbs/breadcrumbs.scss

```diff
@@ -20,28 +20,27 @@
 
 @mixin iui-breadcrumbs-item {
   display: flex;
-  align-items: center;
-
-  > .iui-anchor {
-    display: inline-block;
-  }
+  line-height: $iui-component-height;
+  height: $iui-component-height;
+  max-width: $iui-3xl * 2;
+  margin: 0 $iui-sm;
 
-  > .iui-anchor,
-  > span,
-  .iui-ellipsis {
-    margin: 0 $iui-sm;
+  > .iui-button {
+    margin: 0 (-$iui-s);
```

Comment:
Why negative margin?

Reply:
Button has its own padding, so I'm counteracting the margin that I set above in the breadcrumb-item